db/schema.rb

```diff
@@ -67,6 +67,18 @@
     t.index ["item_id"], name: "index_reviews_on_item_id"
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
**AR Enum**
we can add a column with "role" have the default: 0 
0 - Visito
1 -  Regular User 
2 - Merchant Employee
3 - Admin User